taf/developer_tool.py

```diff
@@ -547,6 +733,37 @@ def _check_key_id(expected_key_id):
     return {"keyid": key_id, "sig": hexlify(signature).decode()}
 
 
+def setup_signing_yubikey(certs_dir=None, scheme=DEFAULT_RSA_SIGNATURE_SCHEME):
+    _, serial_num = yk.yubikey_prompt(
+        "new Yubikey",
+        creating_new_key=True,
+        pin_confirm=True,
+        pin_repeat=True,
+        prompt_message="Please insert the new Yubikey and press ENTER",
+    )
+    _setup_new_yubikey(serial_num, certs_dir)
+
+
+def _setup_new_yubikey(serial_num, certs_dir=None, scheme=DEFAULT_RSA_SIGNATURE_SCHEME):
+    pin = yk.get_key_pin(serial_num)
+    cert_cn = input("Enter key holder's name: ")
+    print("Generating key, please wait...")
+    pub_key_pem = yk.setup(pin, cert_cn, cert_exp_days=EXPIRATION_INTERVAL).decode(
+        "utf-8"
+    )
+    scheme = DEFAULT_RSA_SIGNATURE_SCHEME
+    key = import_rsakey_from_pem(pub_key_pem, scheme)
+    if certs_dir is None:
```

Comment:
Could we move lines 758-766 to a new function, so we can export certificate to `certs_dir` when using existing yubikeys?

Reply:
Done